### PR TITLE
merge accessibility-focused changes into master

### PR DESCRIPTION
**Pull request by atn38**
Lots of small changes:

- add various types of aria-label and role attributes to elements that require it
- reformat headers for coherent structure
- add screen reader only "skip to main content" link
- adjust opacity for background video

and probably other changes I'm not remembering here

### CODE REVIEW

File: public/about.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html><html lang="en"><head>
+<!DOCTYPE html><html lang="en"><head></head><body>f
```
Comment (twhiteaker) [inline public/about.html:1]: <head> shouldn't be closed here.  <body>f also looks weird.

File: public/about.html [hunk 2]
```diff
@@ -27,39 +27,42 @@
...
+   <a id="skip-nav" class="sr-only sr-only-focusable" href="#main-content">
+      Skip Navigation or Skip to Content
+    </a>
       <nav class="navbar navbar-expand-lg navbar-light fixed-top bg-white py-1 px-2">
          <div class="container">
             <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
                <span class="navbar-toggler-icon"></span>
             </button>
-            <a class="navbar-brand" title="BLE Home Page" href="index.html"><img src="img/logo-round-rgb.png" style="max-height: 40px;" alt="BLE home
+            <a class="navbar-brand" title="BLE Home Page" href="index.html" role="button"><img src="img/logo-round-rgb.png" style="max-height: 40px;" alt="BLE home
```
Comment (twhiteaker) [inline public/about.html:42]: Why is role="button" needed here?

File: public/about.html [hunk 3]
```diff
@@ -77,10 +80,10 @@
       </nav>
    </header>
 
-   <div class="container container-small">
+   <div class="container container-small" id="main-content">
```
Comment (twhiteaker) [inline public/about.html:83]: why is this id needed?

File: public/about.html [hunk 4]
```diff
@@ -77,10 +80,10 @@
...
       <h1 class="page-title hyphenate" id="lead">About BLE LTER</h1>
       <br>
-      <figure class="figure m-0">
+      <figure class="figure m-0"  role="none">
```
Comment (twhiteaker) [inline public/about.html:86]: why "none"?

File: public/about.html [hunk 5]
```diff
@@ -123,12 +126,12 @@ <h1 class="page-title hyphenate" id="lead">About BLE LTER</h1>
       <hr>
       <br>
       <h2 id="policies">Policies</h2>
-      <button class="btn btn-lg btn-outline-primary m-1"><a rel="external noopener" target="_blank" href="https://utexas.box.com/v/BLE-DEI-Plan">Diversity, Equity, and Inclusion Plan</a></button>
-      <button class="btn btn-lg btn-outline-primary m-1"><a rel="external noopener" target="_blank" href="https://utexas.box.com/v/ble-authorship-policy">Authorship Policy</a></button>
-      <button class="btn btn-lg btn-outline-primary m-1"><a rel="external noopener" target="_blank" href="https://utexas.box.com/v/ble-data-policy">Data Policy</a></button>
+      <button class="btn btn-lg btn-outline-primary m-1"><a rel="external noopener" target="_blank" href="https://utexas.box.com/v/BLE-DEI-Plan"  aria-label="open diversity equity inclusion plan in new tab">Diversity, Equity, and Inclusion Plan</a></button>
```
Comment (twhiteaker) [inline public/about.html:129]: why is the aria-label needed since the link has text?

File: public/about.html (hunk 3 above)
Comment (twhiteaker) [reply public/about.html:83]: oh I see, so you can skip to here

File: public/bibliography.html [hunk 6]
```diff
@@ -79,10 +82,10 @@
       </nav>
    </header>
 
-   <div class="container">
+   <div class="container" id="main-content">
       <h1 class="page-title hyphenate">BLE LTER Publications</h1>
       <br>
-      <p class="lead">Search our bibliography, or view <a href="https://www.zotero.org/groups/2211939/lter-ble/items" target="_blank" rel="noopener" title="BLE Zotero Group">our library on Zotero</a>.
+      <p class="lead">Search our bibliography, or view <a href="https://www.zotero.org/groups/2211939/lter-ble/items" target="_blank" rel="noopener" title="BLE Zotero Group" aria-label="open in new tab">our library on Zotero</a>.
```
Comment (twhiteaker) [inline public/bibliography.html:88]: shouldn't this say, "open BLE Zotero library in new tab"?

File: public/ims.html [hunk 7]
```diff
@@ -83,13 +86,13 @@
...
+         love collaborating with other information managers on improving the state of IM in LTER. We also make sure to follow LTER guidelines and best practices for <a href="https://lternet.edu/wp-content/uploads/2018/05/Guidelines-for-LTER-Web-Site-Design-and-Content-revised-2018-05-17.pdf" target="_blank" rel="noopener" aria-label="open in new tab">our website</a>, <a href="https://environmentaldatainitiative.org/2018/02/16/eml-best-practices-v-3/" target="_blank" rel="noopener" aria-label="open in new tab">our metadata</a>, and document all our practices in a <a href="https://github.com/BLE-LTER/ble-handbook" target="_blank" rel="noopener">BLE Information Management Handbook</a>. See <a href="https://github.com/BLE-LTER/" target="_blank" rel="noopener" aria-label="open in new tab">what we&apos;re up to on GitHub</a>, and
```
Comment (twhiteaker) [inline public/ims.html:95]: shouldn't the labels say what you're opening in a new tab?  maybe we should view these pages with a screen reader to see what it's like

File: public/index.html [hunk 8]
```diff
@@ -107,37 +110,37 @@ <h1 id="tagline">We study ecosystem change where land meets sea at the top of th
...
       <br>
       <div id="glance">
          <div class="row lead text-white justify-content-center">
             <div class="col-md-5 mb-3 mr-md-3 p-5 text-right" style="background: linear-gradient(rgba(0,0,0,0.4), rgba(0,0,0,0.4)), url(img/12pi.jpg); background-size: cover; background-position: center;">
-               <h1>12</h1>
-               <p class="h3">investigators</p>
-               <p class="h3">from six universities</p>
+               <h3 class="h2">12</h3>
+               <h4 class="h4">investigators</h4>
+               <h4 class="h4">from six universities</h4>
```
Comment (twhiteaker) [inline public/index.html:126]: Did you mean to make the text smaller?

Comment (twhiteaker) [reply public/index.html:126]: Also, these aren't really headings. I think they should all be <p>s with the appropriate heading class.

File: public/about.html (hunk 1 above)
Comment (atn38) [reply public/about.html]: weird. I fixed it here: 249d56c22475755e7aef6362cccb324432e06a37

File: public/index.html (hunk 8 above)
Comment (atn38) [reply public/index.html:126]: what's "s with the appropriate heading class"? do you mean <p>?

File: public/about.html (hunk 2 above)
Comment (atn38) [reply public/about.html:42]: here's the relevant bit from the checker:
image

File: public/about.html (hunk 4 above)
Comment (atn38) [reply public/about.html:86]: here's the relevant bit from the checker
image

File: public/about.html (hunk 5 above)
Comment (atn38) [reply public/about.html:129]: image

File: public/bibliography.html (hunk 6 above)
Comment (atn38) [reply public/bibliography.html:88]: I was doing link-specific labels but then got tired and started doing generic "open in new tab" labels. We can go back and make it specific again.

File: public/about.html (hunk 2 above)
Comment (twhiteaker) [reply public/about.html:42]: Which elements did we get dinged on?  The nav items that truly act as links are fine without the role. The nav links that are just drop down toggles should have role="button".

Comment (atn38) [reply public/about.html:42]: yup, you're correct. Just the drop down toggles, so about, research, and data

File: public/bibliography.html (hunk 6 above)
Comment (twhiteaker) [reply public/bibliography.html:88]: specific seems like the intent of the rule to me

File: public/about.html (hunk 2 above)
Comment (twhiteaker) [reply public/about.html:42]: So for line 42 here, role should be removed.

Comment (atn38) [reply public/about.html]: done at 3c178a14f71b631a502d614ab2edf0b4e02e963e

File: public/index.html (hunk 8 above)
Comment (atn38) [reply public/index.html]: yea, removing the bootstrap class and adding our own (but with essentially the bootstrap class style) gets dinged too. 

my idea: I found out that font-weight seems to be the turnkey, removing it means we're clear. Also, we can get around the multiple `<p>` elements that by themselves don't make sense by adding line breaks, like this: 

```
<p style="font-size: 2rem; line-height: 1.5; margin-bottom: .5rem;">
               12 <br>
               investigators <br>
               from six universities</p>
```
We don't get differentiation between the lines but that's ok with me.

File: public/bibliography.html (hunk 6 above)
Comment (atn38) [reply public/bibliography.html]: more specifics added 8b4d5770e0d587dcdfe0e232bbfd18a1bce3c889

Comment (twhiteaker) [reply public/bibliography.html]: [This site](https://www.deque.com/blog/text-links-practices-screen-readers/) has a The Comparison section which shows how screen readers read such links. Look at row "C. Aria- label present besides anchor text".  Sometimes they read anchor text, sometimes label, sometimes both.  I suppose in this case redundancy is better than an ambiguous link, so I think the way you have it in 8b4d577 is good.

File: public/about.html [hunk 9]
```diff
@@ -41,25 +44,25 @@
...
-                     <a class="nav-link dropdown-toggle" href="#" id="aboutDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">About</a>
+                     <a class="nav-link dropdown-toggle" href="#" id="aboutDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" role="button">About</a>
                      <div class="dropdown-menu" aria-labelledby="aboutDropdownMenuLink">
                         <a class="dropdown-item" href="about.html">Project Overview</a>
                         <a class="dropdown-item" href="people.html">Team</a>
                         <a class="dropdown-item" href="partner.html">Partner Projects</a>
                         <a class="dropdown-item" href="media.html">Media Coverage</a>
                         <div class="dropdown-divider"></div>
-                        <a class="dropdown-item" href="contact.html">Contact Us</a>
+                        <a class="dropdown-item" href="contact.html" role="button">Contact Us</a>
```
Comment (twhiteaker) [inline public/about.html:54]: This one doesn't need a role

File: public/index.html [hunk 10]
```diff
@@ -215,21 +218,21 @@ <h2 id="summers">Open water</h2>
...
+            <h3 class="h3" id="brian">Graduate student spotlight: Brian Kim</h3>
+            <p>The LTER Network Office recently interviewed <a href="student.html#brian">Brian Kim</a> as part of an ongoing LTER graduate student spotlight series. Alongside his research in the sediment biogeochemistry of the lagoons, Brian also serves as the LTER-wide graduate student representative and runs <a href="https://www.instagram.com/lter_community/" target="_blank" aria-label="go to Instagram page">an Instagram account</a> documenting student activities. Read <a href="https://lternet.edu/stories/lter-graduate-student-spotlight-brian-kim/" target="_blank" aria-label="go to publication">his interview here</a> to learn about his research, experience in graduate school, and stories about field work with the BLE team.</p>
```
Comment (twhiteaker) [inline public/index.html:235]: labels should include "in new tab", right?

File: public/index.html [hunk 11]
```diff
@@ -107,37 +110,37 @@ <h1 id="tagline">We study ecosystem change where land meets sea at the top of th
...
       <br>
       <div id="glance">
          <div class="row lead text-white justify-content-center">
             <div class="col-md-5 mb-3 mr-md-3 p-5 text-right" style="background: linear-gradient(rgba(0,0,0,0.4), rgba(0,0,0,0.4)), url(img/12pi.jpg); background-size: cover; background-position: center;">
-               <h1>12</h1>
-               <p class="h3">investigators</p>
-               <p class="h3">from six universities</p>
+               <p style="font-size: 1.75rem; line-height: 1.5; margin-bottom: .5rem;">12 <br>
+               investigators <br>
+               from six universities</p>
```
Comment (twhiteaker) [inline public/index.html:126]: Looks pretty good.  Maybe span the number to increase font size?

File: public/outreach.html [hunk 12]
```diff
@@ -127,16 +130,16 @@ <h1 class="hyphenate border-bottom border-light" id="k12">K-12 Education</h1>
...
                      population on the North Slope. In collaboration with the Bureau of Ocean Energy Management and
                      North Slope Borough Fish and Wildlife Department, we formed a panel of traditional knowledge holders
                      in Kaktovik, Alaska to advise us on emerging ideas and research plans and to act as a conduit for
                   knowledge exchange between BLE and the community.</p>
                   <p>We also share knowledge with local communities in-person through the Barrow Science Fair and
                      Kaktovik Community meetings, in print through our 
-                     <a class="text-white" rel="external noopener" href="https://utexas.box.com/v/BLE-LTER-Kaktovik-newsletter" target="_blank">newsletter The Lagoon Exchange</a>,
-                     and online via our <a class="text-white" rel="external noopener" href="https://facebook.com/arcticlagoons" target="_blank">Facebook page</a>.</p>
+                     <a class="text-white" rel="external noopener" href="https://utexas.box.com/v/BLE-LTER-Kaktovik-newsletter" target="_blank" aria-label="open in new tab">newsletter The Lagoon Exchange</a>,
```
Comment (twhiteaker) [inline public/outreach.html:141]: open The Lagoon Exchange in new tab

File: public/outreach.html [hunk 13]
```diff
@@ -127,16 +130,16 @@ <h1 class="hyphenate border-bottom border-light" id="k12">K-12 Education</h1>
...
+                     and online via our <a class="text-white" rel="external noopener" href="https://facebook.com/arcticlagoons" target="_blank" aria-label="open in new tab">Facebook page</a>.</p>
```
Comment (twhiteaker) [inline public/outreach.html:142]: Open BLE Facebook page in new tab

File: public/outreach.html [hunk 14]
```diff
@@ -168,11 +171,11 @@ <h1 class="hyphenate border-bottom border-light" id="art">Art making</h1>
...
-            <button class="btn btn-link btn-outline-primary m-1"><a rel="external noopener" href="https://twitter.com/ArcticLagoons" target="_blank">Twitter</a></button>
-            <button class="btn btn-link btn-outline-primary m-1"><a rel="external noopener" href="https://utexas.box.com/v/BLE-LTER-Kaktovik-newsletter" target="_blank">The Lagoon Exchange newsletter</a></button>
+            <button class="btn btn-link btn-outline-primary m-1"><a rel="external noopener" href="https://facebook.com/arcticlagoons" target="_blank" aria-label="open in new tab">Facebook</a></button>
+            <button class="btn btn-link btn-outline-primary m-1"><a rel="external noopener" href="https://twitter.com/ArcticLagoons" target="_blank" aria-label="open in new tab">Twitter</a></button>
+            <button class="btn btn-link btn-outline-primary m-1"><a rel="external noopener" href="https://utexas.box.com/v/BLE-LTER-Kaktovik-newsletter" target="_blank" aria-label="open in new tab">The Lagoon Exchange newsletter</a></button>
```
Comment (twhiteaker) [inline public/outreach.html:178]: specify what you're opening in a new tab

File: public/partner.html [hunk 15]
```diff
@@ -83,7 +86,7 @@ <h2 id="aoos">Alaska Ocean Observing System</h2>
...
+            In fall 2020, BLE LTER will begin a partnership with the Alaska Ocean Observing System (AOOS) as an affiliate in AOOS&apos;s next 5-year Intergrated Ocean Observing System plan. This partnership is of key importance in maintaining and expanding our network of moorings across the Beaufort Sea coast. Funding from AOOS will go to instruments recording time series observations of key ecosystems measurements such as pH, temperature, photosynthetically active radiation, and more. See BLE&apos;s time series pH data <a href="https://aoos.org/alaska-ocean-acidification-network/ocean-acidification-data/" target="_blank" aria-label="AOOS site">featured on AOOS&apos;s website</a>.
```
Comment (twhiteaker) [inline public/partner.html:89]: specify what you're opening in a new tab

File: public/student.html [hunk 16]
```diff
@@ -131,7 +135,7 @@ <h1 class="hyphenate" id="julio">Julio Ceniceros</h1>
                support tools, and the development of well-organized adaptation
                strategies for native communities. I&apos;m passionate about keeping our Earth&apos;s environments healthy and love that I get to work in beautiful places like the Arctic and part of the scientific community that shares the same passion. </p>
             <button class="btn btn-outline-primary m-1"><a href="mailto:jeceniceros@miners.utep.edu"><i class="  far fa-paper-plane"></i>  jeceniceros@miners.utep.edu</a></button>
-            <button class="btn btn-outline-primary m-1"><a href="https://storymaps.arcgis.com/stories/c66dc73b0ae04ddfa89e53e58667d834" target="_blank">Research page</a></button>
+            <button class="btn btn-outline-primary m-1"><a href="https://storymaps.arcgis.com/stories/c66dc73b0ae04ddfa89e53e58667d834" target="_blank" aria-label="open page in new tab">Research page</a></button>
```
Comment (twhiteaker) [inline public/student.html:138]: specify what you're opening in new tab

File: public/student.html [hunk 17]
```diff
@@ -194,12 +198,12 @@ <h1 id="danny">Danny Fraser</h1>
...
             <button class="btn btn-outline-primary m-1"><a href="mailto:grifnata@oregonstate.edu"><i class="  far fa-paper-plane"></i>  grifnata@oregonstate.edu</a></button>
-            <button class="btn btn-outline-primary m-1"><a href="https://scholar.google.com/citations?user=0c7X7aMAAAAJ&amp;hl=en" target="_blank">Publications</a></button>
-            <button class="btn btn-outline-primary m-1"><a href="https://natashaagriffin.wordpress.com/" target="_blank">Personal website</a></button>
+            <button class="btn btn-outline-primary m-1"><a href="https://scholar.google.com/citations?user=0c7X7aMAAAAJ&amp;hl=en" target="_blank" aria-label="open page in new tab">Publications</a></button>
+            <button class="btn btn-outline-primary m-1"><a href="https://natashaagriffin.wordpress.com/" target="_blank" aria-label="open page in new tab">Personal website</a></button>
```
Comment (twhiteaker) [inline public/student.html:206]: specify what you're opening in new tab

File: public/student.html [hunk 18]
```diff
@@ -240,11 +244,11 @@ <h1 id="brian">Brian &#xC2B9;&#xD0DD; Kim <span style="font-size: 1rem;">(he/him
...
             <button class="btn btn-outline-primary m-1"><a href="mailto:speterson2@miners.utep.edu"><i class="  far fa-paper-plane"></i>  speterson2@miners.utep.edu</a></button>
-            <button class="btn btn-outline-primary m-1"><a href="https://www.researchgate.net/profile/Sasha_Peterson4/research" target="_blank">Publications</a></button>
+            <button class="btn btn-outline-primary m-1"><a href="https://www.researchgate.net/profile/Sasha_Peterson4/research" target="_blank" aria-label="open page in new tab">Publications</a></button>
```
Comment (twhiteaker) [inline public/student.html:251]: specify what you're opening in new tab

File: public/student.html [hunk 19]
```diff
@@ -306,12 +309,12 @@ <h1 class="hyphenate" id="aqua">Aquanette Sanders</h1>
...
             <p>I am a New Jersey native with educational and research background in marine science and biology. My research interests include ecology and biogeochemistry with a focus on studying rapidly changing coastal systems. My project as part of the BLE integrates high-frequency environmental sensor data with carbon chemistry to model and understand biogeochemical and ecological processes occurring in the Beaufort lagoons and nearby aquatic systems.</p>
             <p>I am the current BLE student representative, advocating for the interests of all BLE-affiliated students.</p>
             <button class="btn btn-outline-primary m-1"><a href="mailto:aspera@miners.utep.edu"><i class="  far fa-paper-plane"></i>  aspera@miners.utep.edu</a></button>
-            <button class="btn btn-outline-primary m-1"><a href="https://orcid.org/0000-0001-5025-1260" target="_blank">Publications</a></button>
+            <button class="btn btn-outline-primary m-1"><a href="https://orcid.org/0000-0001-5025-1260" target="_blank" aria-label="open page in new tab">Publications</a></button>
```
Comment (twhiteaker) [inline public/student.html:317]: specify what you're opening in new tab

File: public/student.html [hunk 20]
```diff
@@ -331,11 +334,11 @@ <h1 class="hyphenate" id="alina">Alina Spera <span style="font-size: 1rem;">(she
...
             <p>My doctoral work focuses on aquatic systems and trophic linkages that shape food web form and function. I am interested in determining what energy sources (e.g., terrestrial, bacterial, phytoplanktonic) fuel food webs and how these sources change through time and space. I use biomarkers (e.g., stable isotopes, fatty acids) as tools to better understand trophic connectedness and resilience of food webs. It is important to me to work in environments experiencing rapid change to better understand the implications of climate change at a global scale.</p>
             <button class="btn btn-outline-primary m-1"><a href="mailto:swilkinson4@alaska.edu"><i class="  far fa-paper-plane"></i>  swilkinson4@alaska.edu</a></button>
-            <button class="btn btn-outline-primary m-1"><a href="https://orcid.org/0000-0002-5244-7213" target="_blank">Publications</a></button>
+            <button class="btn btn-outline-primary m-1"><a href="https://orcid.org/0000-0002-5244-7213" target="_blank" aria-label="open page in new tab">Publications</a></button>
```
Comment (twhiteaker) [inline public/student.html:341]: specify what you're opening in new tab

File: templates/template.html [hunk 21]
```diff
@@ -42,27 +45,27 @@
...
-                        aria-haspopup="true" aria-expanded="false">About</a>
+                        aria-haspopup="true" aria-expanded="false" role="button">About</a>
                      <div class="dropdown-menu" aria-labelledby="aboutDropdownMenuLink">
                         <a class="dropdown-item" href="about.html">Project Overview</a>
                         <a class="dropdown-item" href="people.html">Team</a>
                         <a class="dropdown-item" href="partner.html">Partner Projects</a>
                         <a class="dropdown-item" href="media.html">Media Coverage</a>
                         <div class="dropdown-divider"></div>
-                        <a class="dropdown-item" href="contact.html">Contact Us</a>
+                        <a class="dropdown-item" href="contact.html" role="button">Contact Us</a>
```
Comment (twhiteaker) [inline templates/template.html:55]: role not needed

Comment (atn38) [reply templates/template.html]: fixed in e42d2addcf2e53fd2b019d3b75afb3cc9fab9635

File: public/index.html (hunk 11 above)
Comment (atn38) [reply public/index.html]: fixed in e42d2addcf2e53fd2b019d3b75afb3cc9fab9635

File: public/about.html (hunk 9 above)
Comment (atn38) [reply public/about.html]: fixed in e42d2addcf2e53fd2b019d3b75afb3cc9fab9635

File: public/student.html (hunk 20 above)
Comment (atn38) [reply public/student.html]: fixed in c4d830a7a3cf15b16021792b9e4da7d42427b8c5

File: public/student.html (hunk 19 above)
Comment (atn38) [reply public/student.html]: fixed in c4d830a7a3cf15b16021792b9e4da7d42427b8c5

File: public/student.html (hunk 18 above)
Comment (atn38) [reply public/student.html]: fixed in c4d830a7a3cf15b16021792b9e4da7d42427b8c5

File: public/student.html (hunk 17 above)
Comment (atn38) [reply public/student.html]: fixed in c4d830a7a3cf15b16021792b9e4da7d42427b8c5

File: public/outreach.html (hunk 14 above)
Comment (atn38) [reply public/outreach.html]: fixed in c4d830a7a3cf15b16021792b9e4da7d42427b8c5

File: public/outreach.html (hunk 13 above)
Comment (atn38) [reply public/outreach.html]: fixed in c4d830a7a3cf15b16021792b9e4da7d42427b8c5

File: public/outreach.html (hunk 12 above)
Comment (atn38) [reply public/outreach.html]: fixed in c4d830a7a3cf15b16021792b9e4da7d42427b8c5

File: public/index.html (hunk 10 above)
Comment (atn38) [reply public/index.html]: fixed in c4d830a7a3cf15b16021792b9e4da7d42427b8c5

File: public/index.html [hunk 22]
```diff
@@ -215,21 +218,21 @@ <h2 id="summers">Open water</h2>
...
+            <h3 class="h3" id="jim">New coastal permafrost research shows unexpected results</h3>
+            <p>A new publication, co-authored by BLE LTER investigator James McClelland, uses subsurface mapping to show permafrost to be mostly absent under the shallow seafloor in Kaktovik Lagoon. This research poses implications on sea-land water exchange, carbon dynamics, and coastal erosion. Read <a href="https://advances.sciencemag.org/content/6/43/eabb5083" target="_blank" aria-label="go to publication">the paper on Science Advances</a> or <a href="https://www.jsg.utexas.edu/news/2020/10/coastal-permafrost-more-susceptible-to-climate-change-than-previously-thought/" target="_blank" aria-label="go to publication">coverage</a> by The University of Texas Jackson School of Geosciences.</p>
```
[the remark this answers is not in the document]
Comment (atn38) [reply public/index.html]: fixed in c4d830a7a3cf15b16021792b9e4da7d42427b8c5

**Pull request merged**